.github/workflows/ci.yaml

```diff
@@ -20,4 +20,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - run: docker build --file e2e2/test/images/Dockerfile.aws-efa-nccl-tests .
\ No newline at end of file
+    - run: docker build --file e2e2/test/images/Dockerfile.aws-efa-nccl-tests .
+  build-bert-training:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - run: docker build --file e2e2/test/images/bert-training/Dockerfile.bert-training e2e2/test/images/bert-training
```

Comment:
an idea for the future (since there will be more to come), maybe we standardizing a `images/XXX/{Dockerfile, ...}` structure for all our images and then create a job matrix for test image build